scripts/funcotator/data_sources/getGencode.sh

```diff
@@ -158,25 +162,35 @@ function getGencodeFiles()
 
     mkdir -p ${OUT_DIR_NAME}/${refVersion}
     pushd ${OUT_DIR_NAME}/${refVersion} &> /dev/null
-    wget ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_${version}/gencode.v${version}.annotation.gtf.gz
-    wget ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_${version}/gencode.v${version}.pc_transcripts.fa.gz
 
-    gunzip gencode.v${version}.annotation.gtf.gz
-    gunzip gencode.v${version}.pc_transcripts.fa.gz
+		fileRefVersion=${version}
+		if [[ "${refVersion}" == "hg38" ]] ; then
+			sourceUrl=ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_${version}/gencode.v${version}.annotation.gtf.gz
+			wget ${sourceUrl} 
+    	wget ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_${version}/gencode.v${version}.pc_transcripts.fa.gz
+		else
+			fileRefVersion="${version}lift37"
+			sourceUrl=ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_${version}/GRCh37_mapping/gencode.v${fileRefVersion}.annotation.gtf.gz
+			wget ${sourceUrl}
+			wget ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_${version}/GRCh37_mapping/gencode.v${fileRefVersion}.pc_transcripts.fa.gz
+		fi
+		
+    gunzip gencode.v${fileRefVersion}.annotation.gtf.gz
+    gunzip gencode.v${fileRefVersion}.pc_transcripts.fa.gz
 
     # We must fix the information in the gencode gtf file:
     echo "Reordering Gencode GTF data ..."
-    ${SCRIPTDIR}/fixGencodeOrdering.py gencode.v${version}.annotation.gtf > gencode.v${version}.annotation.REORDERED.gtf
+    ${SCRIPTDIR}/fixGencodeOrdering.py gencode.v${fileRefVersion}.annotation.gtf | sed -e '1,5s$^#\([a-z]\)$##\1$' > gencode.v${fileRefVersion}.annotation.REORDERED.gtf
 
     # Clean up original file:
-    rm gencode.v${version}.annotation.gtf
+    rm gencode.v${fileRefVersion}.annotation.gtf
 
     echo "Creating config file ..."
-    createConfigFile "${DATA_SOURCE_NAME}" "${version}" "gencode.v${version}.annotation.REORDERED.gtf" "ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_${version}/gencode.v${version}.annotation.gtf.gz" "gencode.v${version}.pc_transcripts.fa" > gencode.config
+    createConfigFile "${DATA_SOURCE_NAME}" "${version}" "${refVersion}" "gencode.v${fileRefVersion}.annotation.REORDERED.gtf" "${sourceUrl}" "gencode.v${fileRefVersion}.pc_transcripts.fa" > gencode.config
 
     if $HAS_SAMTOOLS ; then
-        echo "Indexing Fasta File: gencode.v${version}.pc_transcripts.fa"
-        samtools faidx gencode.v${version}.pc_transcripts.fa
+        echo "Indexing Fasta File: gencode.v${fileRefVersion}.pc_transcripts.fa"
+        samtools faidx gencode.v${fileRefVersion}.pc_transcripts.fa
```

Comment:
Do you create the `.dict` file as well?

Reply:
No.  I print to the user at the end that they need to make their own dict and fasta index file (lines 252-274).

I don't have a way of guaranteeing that the GATK is on their path to create the sequence dictionary (or samtools for that matter).